### PR TITLE
Customize wording for art show application confirmation

### PR DESCRIPTION
**Pull request by kitsuta**
Fixes https://github.com/MidwestFurryFandom/art_show/issues/67.

### CODE REVIEW

**Comment by kitsuta** [comment]
It references "November 1st" so it is effectively hard coding an MFF specific date in the email.

File: mff_rams_plugin/templates/emails/art_show/application.html [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+Should your application be approved, you will be required to pay for your space to complete your reservation.
+{% if app.attendee.badge_status == c.NEW_STATUS %}
+You MUST complete your {{ c.EVENT_NAME }} registration before you will be able to pay for your art show space. You
+may complete your registration
+<a href="{{ c.URL_BASE }}//preregistration/confirm?id={{ app.attendee_id }}">here</a>.
+{% endif %}
+
+{% if app.agent_code %}
+<br/><br/>Please provide your Agent with the Agent registration code <strong>{{ app.agent_code }}</strong> so that they may register
```
Comment (kitsuta) [inline mff_rams_plugin/templates/emails/art_show/application.html]: To clarify, this is the exact bit of text that's MFF-specific.

**Comment by jmdawson** [comment]
I think we established that this does not reflect the addition of the General vs. Adult panel & table space, so this needs to be updated before we can merge it, or else it will hide the additions in art_show

**Comment by kitsuta** [comment]
Done!

**Pull request merged**